### DISCUSSION TITLE
Performance drop in last update

### DISCUSSION THREAD

**Discussion by motorist828** [vladmandic/sdnext#674] [Q&A] [answered]
I noticed a twofold drop in the generation rate 
It seems to be related to the update that came out today
in the morning it worked well (every day I do a git pull)
I'm not sure if this is a bug, so I'm writing here

![Screenshot_119](https://user-images.githubusercontent.com/40582999/235501208-6b670c26-af2a-4116-944b-11ec89496479.png)
![Screenshot_120](https://user-images.githubusercontent.com/40582999/235501590-bbaf530c-f23c-4fd4-83bc-ead62ffda86e.png)

**Comment by vladmandic** [comment, marked as answer]
yup, known, recent code change upstream is the cause and i've notified the author:
<https://github.com/KohakuBlueleaf/a1111-sd-webui-lycoris/issues/21>

as a temporary workaround, you can disable the extension `a1111-sd-webui-lycoris`

**Reply by motorist828** [reply]
Thanks for the quick response

> as a temporary workaround, you can disable the extension `a1111-sd-webui-lycoris`

How to do it right?

**Reply by vladmandic** [reply]
1. UI -> Settings -> Extensions -> unselect checkbox next to extension
2. Restart server from command line so extension is not loaded during startup.
